SayTheirNames/Source/View/Home/Person/PersonCell.xib

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
same here

Reply:
Merging with latest develop resolved this

Reply:
This file is still here 😅